File: C/0657.c
bool judgeCircle(char* moves) {
    int x = 0, y = 0;
    int len = strlen(moves);

    for (int i = 0; i < len; i++) {
        if (moves[i] == 'U') {
            y++;
        } else if (moves[i] == 'D') {
            y--;
        } else if (moves[i] == 'R') {
            x++;
        } else {
            x--;
        }
    }

    return (x == 0 && y == 0);
}

File: Python/0657.py
class Solution:
    def judgeCircle(self, moves: str) -> bool:
        position = [0, 0]

        for move in moves:
            if move == 'U':
                position[1] += 1
            if move == 'D':
                position[1] -= 1
            if move == 'R':
                position[0] += 1
            if move == 'L':
                position[0] -= 1

        return position == [0, 0]
